platform/src/test/kotlin/org/stellar/anchor/platform/PlatformRateIntegrationTest.kt

```diff
@@ -33,14 +31,6 @@ class PlatformRateIntegrationTest {
     rateIntegration = PlatformRateIntegration(server.url("").toString(), OkHttpUtil.buildClient())
   }
 
-  private fun getRateResponse(price: String, expiresAt: LocalDateTime?): MockResponse {
-    var expiresAtStr: String? = null
-    if (expiresAt != null) {
-      expiresAtStr = expiresAt.format(DateTimeFormatter.ISO_DATE_TIME)
-    }
-    return getRateResponse(price, expiresAtStr)
-  }
-
   private fun getRateResponse(price: String, expiresAt: String? = null): MockResponse {
```

Comment:
Can we move these tests to: 

https://github.com/stellar/java-stellar-anchor-sdk/blob/main/platform/src/main/kotlin/org/stellar/anchor/platform/Sep38Tests.kt

Reply:
Started adding tests for them in https://github.com/stellar/java-stellar-anchor-sdk/pull/147.